### ISSUE TITLE
operating system in sentry issue is the server OS not browser OS

### ISSUE THREAD

**Issue by runawaycoin** [getsentry/sentry-php#826]
Hi, 

In version 2.1.0, the reported OS details to sentry is for the server. Previously wasn't this the client (browser) and parsed from the user-agent?

Its confusing in sentry dashboard as the browser name and version are shown alongside the server OS - where the browser OS would be more useful.

![image](https://user-images.githubusercontent.com/26024569/58908510-aa68e500-8708-11e9-95d4-d677714a6bd4.png)

**Comment by ste93cry** [comment]
I agree that the way the information are presented to the user is confusing, however for a server-side library reporting the OS of the server is way more useful than the OS of the client or the browser information imho

**Comment by JanMikes** [comment]
Hello, i just noticed same issue. After moving from Raven handler i noticed the difference.

Though i have to admit that @ste93cry is right, for PHP client's os does not really matter (it does only in terms of javascript/client side language).

**Comment by ste93cry** [comment]
I would say that this is just a UX issue, since we are effectively sending both the `User-Agent` header and the information of the server OS, so the Sentry UI should be capable of distinguish them and show them appropriately. @HazAT maybe this issue should be moved to [getsentry/sentry](https://github.com/getsentry/sentry) and further discussed there in terms of UI changes?
